### PR TITLE
[DNM]Add prices to a lot of cosmetic items. Misc. pricing change [WIP]

### PR DESCRIPTION
**Pull request by Anasari**
Changelog:
All AutoDrobe items cost money, baseline is $100
High demand items like Kitty ears & School girl uniform = $500
Zippo lighter now costs $200, as befitting a high-class, proper lighter
Price of suitlord, shoelord & hatlord is now $50 per item
Price of pet collar is $500. Putting them on your co-worker is now costly. Or reserved for the HoP

EDIT1: Note - In their vendor
EDIT2: Time constraint. Please DNM at this stat.

:cl:Ansari
tweak: Cosmetic items in locker room cost money. Misc. price change.
/:cl:

### CODE REVIEW

**Comment by DesolateG** [comment]
I'll assume this is a meme PR for #8200 ?

**Comment by taukausanake** [comment]
It shouldn't be. Why would you have a clothing vendor, or ANY vending machine, just pop out items for free? If anything this should have been done a long time ago

**Comment by tigercat2000** [comment]
Not a meme PR. Can't say I approve much, though, 200 credits is a substantial cost for silly outfits that people use to make their characters look better. It also gives an objective advantage to the loadout system, as those items don't cost money- and they shouldn't.

**Comment by TDSSS** [comment]
Needs the meme PR label

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -921,6 +921,7 @@
 	density = 0 //It is wall-mounted, and thus, not dense. --Superxpdude
 	products = list(/obj/item/stack/medical/bruise_pack = 2,/obj/item/stack/medical/ointment = 2,/obj/item/weapon/reagent_containers/hypospray/autoinjector = 4,/obj/item/device/healthanalyzer = 1)
 	contraband = list(/obj/item/weapon/reagent_containers/syringe/charcoal = 4,/obj/item/weapon/reagent_containers/syringe/antiviral = 4,/obj/item/weapon/reagent_containers/food/pill/tox = 1)
+	prices = list(/obj/item/stack/medical/bruise_pack = 20,/obj/item/stack/medical/ointment = 50,/obj/item/weapon/reagent_containers/hypospray/autoinjector = 1000,/obj/item/device/healthanalyzer = 20,/obj/item/weapon/reagent_containers/food/pill/tox = 10)
```
Comment (SamHPurp) [inline code/game/machinery/vending.dm]: No on this.

God damn American health care systems.

**Comment by Anasari** [comment]
I've made some mistake and it seems the PR has been mistaken for a meme PR by some people.

I'll add in more pricing for hacked items that is free in hacked-machine-with-actual-price.

For tonight (HKT) I will just do some changes to fix some mistakes and respond to tiger's comment.

I believe that the loadout advantage won't be substantial.

**Comment by Anasari** [comment]
Autodrobe item changed to $100.
Kitty ears & schoolgirl uni price changed to $500, as they have a high demand.
Removed nanomed change.

**Comment by Ty-Omaha** [comment]
Before we go ahead and make everything cost something, can we _please_ get a way to make money _**first**_? 

I wouldn't mind if you still reliably got money from slot machines but that is not the case anymore.

**Comment by alexkar598** [comment]
^^^^^^

**Comment by taukausanake** [comment]
> It also gives an objective advantage to the loadout system

Then certain items shouldn't be in the loadout system when they are already in the vendors. That would be my argument at least. Still would be nice if karma was used for loadout in some way. Would make more sense as something that needs to be earned

**Comment by uraniummeltdown** [comment]
Hell no, the wacky Autodrobe clothing is great for disguises. All the fun clothes are there, and anyway clothing is in limited stock. Having to worry about money too sucks.

**Comment by ktoma36** [comment]
Considering how the money you start with is completely random, I have joined in roundstart as a clown with only 140$ before, putting price tags on things that are purely cosmetic seems kinda silly. Especially since there is no reliable way to earn money as @Ty-Omaha said.

There are only four ways I can think of for someone to earn money right now
- Slots
- Traitor item briefcase full of cash
- Mining and using points to get cash
- Service borg dispensing money (in 1$ increments)

**Comment by Ty-Omaha** [comment]
@ktoma36 I wouldn't count slots in that list anymore:
https://github.com/ParadiseSS13/Paradise/pull/8182

**Comment by Fox-McCloud** [comment]
Going to echo @tigercat2000 and @uraniummeltdown meltdown here.

**Comment by SamHPurp** [comment]
Going to muffle @Fox-McCloud echo here. ;)

Now that medbay stuff is removed, I'm cool with this.

**Comment by Anasari** [comment]
I believe Ty has a good point that there isn't sufficient method to earn money for your regular crewmembers. Although I believe that even with the amount of people start on average, people should be able to afford basic clothings.

I will work on a system that provide income for people in the future.

**Comment by DesolateG** [comment]
Wouldn't it be a better approach to make an income system and then add prices to things? 

It's already partially there; robotics gets dosh for building cyborgs and ripleys, science gets it for turning in research disks on the cargo shuttle, and miners can turn in points for it.

Iirc there's also something out there, possibly Goon that allows engineering to earn money off of excess power produced.

**Comment by taukausanake** [comment]
@DesolateG a problem with that is that not many people, at least in my opinion, don't do the shipping things because they don't need the money for anything. They don't spend enough of it to warrant getting any. Adding a price to the vendors might change that.

Idea: Add a new mechanic, maybe an ID unlock on the circuit board to vending machines that allows for making things free. Like the R&D console. Make it so HoP and/or QM are the only ones who can unlock it. I'd get behind that

**Comment by ktoma36** [comment]
If prices are added to the clothes can the stock number in the machine be changed? The limited stock feature is already kinda mute considering anyone with tools can reset the stock by deconstructing and reconstructing the machine.

**Comment by Rumiluntti** [comment]
The pet collar is way too cheap.
Kitty ears still don't cause brain damage to the wearer so they should have a few zeroes added to the price too.

**Comment by Jountax** [comment]
Bartender is allowed to charge for drinks, but that's not really "making" money.

Scientists all get paid when they send tech levels back to CC though, and I think roboticists get some money when they make a borg.

**Comment by Dumbdumn5** [comment]
I'm of the mindset that we need a way to make money first and foremost before we have a way to lose it regularly.  As both Tiger, Fox, and uranium have said, it's a system that's already limited enough by only having one of each item in addition to giving an objective advantage in terms of aesthetics to a paid loadout system due to the expansion of loadout points. Money has to be made before it can be spent. :-1:

**Pull request closed without merging**